### DISCUSSION TITLE
Identify Xiaomi Mi Robot Vacuum-Mop P

### DISCUSSION THREAD

**Discussion by emmexx** [Hypfer/Valetudo#1963] [Q&A / Support] [answered]
As suggested in the [supported models page](https://valetudo.cloud/pages/general/supported-robots.html#xiaomi_vacuummop_p) I ask here to be sure before rooting.
I own a Mi Robot Vacuum-Mop P white version.
In the label it is defined as
Name: **Mi Robot Vacuum-Mop P (Smart Vacuum)**
Model: **STYTJ02YM**
Manufactured by: **Foshan VIOMI Electrical Technology Co., Ltd.**
I don't see any reference to a v8 version in the label.

There's only a micro usb connector under the battery.

Apart from the color, my robot is almost identical to the one on the pictures found in the supported models page.
Can I root it?
Can I check something else to be sure?

**Comment by stefanmd023** [comment, marked as answer]
what is the wifi ap name it generates when you reset the wifi?

**Reply by emmexx** [reply]
viomi-vacuum-v8_miapB3E7 :-(
So it's better not rooting it?

**Reply by stefanmd023** [reply]
it is definitely very high risk

**Reply by emmexx** [reply]
There's some hope that the issue will be solved some day or I'd better forget about Valetudo?

**Reply by stefanmd023** [reply]
if anyone is working on it they did not make it public afaik, so don't hold your breath
